components/ide/code-desktop/supervisor-ide-config_insiders.json

```diff
@@ -0,0 +1,11 @@
+{
+    "entrypoint": "/ide-desktop/startup.sh",
+    "entrypointArgs": [ "Open in VS Code Insiders on Desktop", "vscode-insiders" ],
```

Comment:
**suggestion**: Although I appreciate the accuracy here, I still think this could be simplified as mentioned in https://github.com/gitpod-io/gitpod/pull/6270#discussion_r734679322 but do not want to go back and forth here. I'd like to suggest once more going with something like _Open in XYZ_ but I'll leave this to you to decide.

**thought**: Also, happy to discuss this in a new follow up issue or https://github.com/gitpod-io/gitpod/issues/6602. Maybe updating the design here to be less verbose or using different elements could help facilitate more verbose options like _Open in VS Code Insiders on Desktop_. Cc @loujaybee  🏓 

In this case it would become _Open in VS Code Insiders_ while the secondary option would be :a: _Open in Browser_ or  :b: _Open in Gitpod_. See also https://github.com/gitpod-io/gitpod/pull/6270#discussion_r735492810.

| BEFORE | AFTER :a: | AFTER :b: |
|-|-|-|
| <img width="1440" alt="Screenshot 2021-11-18 at 1 16 17 AM (2)" src="https://user-images.githubusercontent.com/120486/142298110-6781bc98-aa0b-47f6-afd8-e05be5949861.png"> | <img width="1440" alt="Screenshot 2021-11-18 at 1 18 17 AM (2)" src="https://user-images.githubusercontent.com/120486/142298098-0490099b-93e1-4716-a4c9-e55a32dfeec6.png"> | <img width="1440" alt="Screenshot 2021-11-18 at 1 16 35 AM (2)" src="https://user-images.githubusercontent.com/120486/142298099-b3d7d1ab-855e-41c7-8d45-eec1a7f52c47.png"> |

Reply:
I like A

Reply:
→ https://github.com/gitpod-io/gitpod/issues/6769